xapian-core/backends/database.h

```diff
@@ -149,6 +149,13 @@ class Database::Internal : public Xapian::Internal::intrusive_base {
 	 */
 	virtual	Xapian::termcount get_unique_terms(Xapian::docid did) const = 0;
 
+	/** Get the max wdf of terms in document.
+	 *
+	 *  @param did  The document id of the document whose number of terms is
+	 *		being requested.
```

Comment:
`number of terms` here is wrong (presumably due to cut+paste without changing enough).